.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
I thought that the vars didn't need to be specified if unchanged, but for some reason I was under the impression that had caused failures in the past. I don't have proof of them, though, so I'll try again.

I take your point that there shouldn't be an environment for every single minor version of every dependency. This one is meant to be temporary, too.

With that in mind, I'll rewrite some of this to maybe only have one, such that updating it is as simple as removing a travis env and updating the version number, which is a combination of what you've already proposed.